tests/integration/scale_test.go

```diff
@@ -34,7 +34,23 @@ func scaleTests(t *testing.T, when spec.G, it spec.S) {
 			assert.Equal(t, 0, service.GetAvailableReplicas())
 			assert.Equal(t, 0, service.GetScale())
 		})
-		it.Pend("should scale to x,y,z....", func() {
+		it("should scale to 3", func() {
+			assert.Equal(t, 1, service.GetAvailableReplicas())
+			service.Scale(3)
+			assert.Equal(t, 3, service.GetAvailableReplicas())
+			assert.Equal(t, 3, service.GetScale())
+		})
+		it("should scale to 5", func() {
+			assert.Equal(t, 1, service.GetAvailableReplicas())
+			service.Scale(5)
+			assert.Equal(t, 5, service.GetAvailableReplicas())
+			assert.Equal(t, 5, service.GetScale())
+		})
+		it("should scale to 10", func() {
+			assert.Equal(t, 1, service.GetAvailableReplicas())
+			service.Scale(10)
+			assert.Equal(t, 10, service.GetAvailableReplicas())
+			assert.Equal(t, 10, service.GetScale())
```

Comment:
On my k3s cluster, tests for setting scale above 2 were a bit flaky. I had to update /tests/testutil/service.go in the `waitForScale` function to wait for 180 seconds at most (line 261)

Reply:
Added fix for that here: https://github.com/rancher/rio/pull/552/files#diff-ebf1c8ea2e75e3983bbc0d00d1ed73e0R240

In short: it will wait `scale * 20 seconds`, or 2 minutes, whichever is greater. In this case it will wait 200 seconds. May need to bump that up to 25 or 30 seconds depending on systems we are targeting.